onnxruntime/test/onnx/main.cc

```diff
@@ -286,7 +287,18 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
     fwrite(res.c_str(), 1, res.size(), stdout);
   }
   // clang-format off
-  std::map<std::string, std::string> broken_tests{
+
+  struct BrokenTest
+  {
+    std::string test_name_;
+    std::string reason_;
+    std::set<int64_t> broken_versions_ = {}; // apply to all versions if empty
+    bool operator < (const struct BrokenTest& test) const {
+        return strcmp(test_name_.c_str(), test.test_name_.c_str()) < 0;
+    }
+  };
+
+  std::set<BrokenTest> broken_tests = {
```

Comment:
>std::set<BrokenTest [](start = 2, length = 19)

Has Pranav talked to you about MVN test?

Reply:
Yes, added in disabling list.